content/install-hook-openusercss.js

```diff
@@ -0,0 +1,159 @@
+'use strict';
+
+const manifest = chrome.runtime.getManifest();
+const allowedOrigins = [
+  'https://openusercss.org',
+  'https://openusercss.com'
+];
+
+const askHandshake = () => {
+  // Tell the page that we exist and that it should send the handshake
+  allowedOrigins.forEach(origin => {
+    window.postMessage({
+      'type': 'ouc-begin-handshake'
+    }, origin);
+  });
+};
+
+// Listen for queries by the site and respond with a callback object
+const sendInstalledCallback = styleData => {
+  allowedOrigins.forEach(origin => {
+    window.postMessage({
+      'type':  'ouc-is-installed-response',
+      'style': styleData
+    }, origin);
+  });
+};
+
+const attachInstalledListeners = () => {
+  window.addEventListener('message', event => {
+    if (
+      event.data
+      && event.data.type === 'ouc-is-installed'
+      && allowedOrigins.includes(event.origin)
+    ) {
+      chrome.runtime.sendMessage({
+        'method':    'findUsercss',
+        'name':      event.data.name,
+        'namespace': event.data.namespace
+      }, style => {
+        if (style) {
+          sendInstalledCallback({
+            'installed': true,
+            'enabled':   style.enabled,
+            'name':      event.data.name,
```

Comment:
Make it DRY by using shorthand initializers for `name` and `namespace`. Or maybe make one object for the message, change the relevant properties, then send it using just one sendInstalledCallback call.